frontend/src/layout/navigation-3000/sidepanel/sidePanelStateLogic.tsx

```diff
@@ -46,4 +64,53 @@ export const sidePanelStateLogic = kea<sidePanelStateLogicType>([
             }
         },
     })),
+
+    urlToAction(({ actions, values }) => ({
+        '*': (_, _search, hashParams) => {
+            if ('supportModal' in hashParams) {
+                const [kind, area] = (hashParams['supportModal'] || '').split(':')
+
+                delete hashParams['supportModal'] // legacy value
```

Comment:
More general question. I've seen us refer to hash params as "legacy" in a couple of places. Is there a rationale to deprecate them? Is everything just moving to query params?

Reply:
We moved to query params for the session recordings but thats because it is generally typical to the page in question.

In this case me calling "legacy" is more about the `supportModal` than the use of a hash param. For the panel I think it makes total sense as it is tangental to whatever the main page content is